src/EIPTestsFiller/StateTests/stEOF/stEIP3540/CREATE2_EOF1InvalidFiller.yml

```
@@ -402,23 +402,10 @@ CREATE2_EOF1Invalid:
             nonce: 1
             storage:
               1 : '1'
-      # Not bumping nonce
-      - indexes:
-          data:
-            - '0-51'
-        network:
-          - '>=Cancun'
-        result:
-          a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
-            nonce: 1
-          b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
-            nonce: 0
-            storage:
-              1 : '1'
       # Bumping nonce
       - indexes:
```

As we are bumping the nonce for all the tests, we don't need to specify the indexes of each transaction, we can let the indexes section as follows:

```suggestion
      - indexes: {}
```

---

Done

---

BTW. It looks that expect section in `CREATE_EOF1Invalid` and its counterpart from `CREATE2` look the same for Cancun and Shanghai. Is it correct?

---

I think it's OK. Originally we had the same expectation for both forks. But for the fork containing EOF we had this comment `# Same expectation, but for ~~London~~Shanghai execution fails on EF byte, for Cancun it fails because EOF initcode or code is invalid"`, maybe we can add this instead of the `#Bumping nonce` comment.

---

done